dataflux_pytorch/_helper.py

```diff
@@ -30,7 +31,7 @@ def _get_missing_permissions(storage_client: any, bucket_name: str,
 
     try:
         perm = bucket.test_iam_permissions(required_perm)
-    except Exception as e:
-        logging.exception(f"Error testing permissions: {e}")
+    except Exception:
+        raise
```

Comment:
No need to have the try/except block here now, it's just the same as doing this outside of the block and letting the exception raise itself

Reply:
Reverted the changes I made to the other two files and updated the exception message here. Stack trace now looks like this: http://shortn/_T9BGgSybjP